### DISCUSSION TITLE
Can i join team sir

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#18319] [General]
Git ID deadaf5

DT
https://github.com/deadaf5/android_xaiomi_lancelot/tree/spark
https://github.com/deadaf5/mt6768-common

Successful build
https://cirrus-ci.com/build/4612399706144768

**Comment by Apon77** [comment]
please follow #learn instruction from telegram group

**Reply by hipexscape** [reply]
Sir what I am missing
